### PR TITLE
#1 nan seconds ago

### PR DESCRIPTION
**Pull request by nesyG**
I managed to get the correct values for reviews. 
The getTimeAgo function worked as expected, the problem was likely in the rendering component.
Let me know if this would be an acceptable solution

### CODE REVIEW

File: frontend/src/pages/MovieDetails.tsx [hunk 1]
```diff
@@ -80,7 +84,7 @@ export default function MovieDetails() {
         setUserReview({
           ...reviewPayload,
           edited: false,
-          createdAt: "",
+          createdAt: getTimeAgo(new Date().toISOString()),
```
Comment (aybeedee) [inline frontend/src/pages/MovieDetails.tsx]: Just replacing the "" with new Date().toISOString() here seems to be enough to resolve the issue without requiring changes in any other place. So this change would read as
```
createdAt: new Date().toISOString(),
```

**Review by aybeedee — CHANGES_REQUESTED** [review]
Hi, thanks for the contribution. Solution is along the right lines. You can view the comment on the change, update it accordingly and then we can merge it in. Thanks!

File: frontend/src/pages/MovieDetails.tsx (hunk 1 above)
Comment (nesyG) [reply frontend/src/pages/MovieDetails.tsx]: Oh that's right, tnx. I just pushed adjusted version!

**Review by aybeedee — APPROVED** [review]

**Comment by aybeedee** [comment]
looks good to me, thanks

**Pull request merged**